src/components/TransactionItemRow/index.tsx

```diff
@@ -195,9 +201,9 @@ function TransactionItemRow({
                 <View style={[StyleUtils.getReportTableColumnStyles(CONST.SEARCH.TABLE_COLUMNS.FROM)]}>
                     {!!transactionItem.to && (
                         <UserInfoCell
-                            accountID={transactionItem.to.accountID}
-                            avatar={transactionItem.to.avatar}
-                            displayName={transactionItem.to.displayName ?? ''}
+                            accountID={transactionItem?.to.accountID}
```

Comment:
At the top, we already have a condition that checks if to exists, so I'm not sure if ? is necessary here.

                   {!!transactionItem.to && (